docs/INSTALLATION.md

```diff
@@ -1,18 +1,21 @@
 # Installing Delphi and DUnitX Testing Framework #
 ### Windows ###
 
-30 day free trial of Embarcadero's RAD Studio or Delphi may be downloaded from [here](https://www.embarcadero.com/products?utm_source=google&utm_medium=cpc&utm_campaign=brand&utm_content=brand&utm_embarcadero&gclid=CjwKEAiAp97CBRDr2Oyl-faxqRMSJABx4kh9V8bOEuG0CznQ9AGToIyuKeTzvevljmHTboYXk4n6OxoC4Frw_wcB).  Follow installation instructions included with the product.
+A 30 day free trial of Embarcadero's RAD Studio or Delphi may be downloaded from [here](https://www.embarcadero.com/products?utm_source=google&utm_medium=cpc&utm_campaign=brand&utm_content=brand&utm_embarcadero&gclid=CjwKEAiAp97CBRDr2Oyl-faxqRMSJABx4kh9V8bOEuG0CznQ9AGToIyuKeTzvevljmHTboYXk4n6OxoC4Frw_wcB).  Alternatively, a free [Starter Edition](https://www.embarcadero.com/products/delphi/starter/promotional-download) version is also available.  Follow installation instructions included with the product.
 
 ### Non-Windows based Operating Systems ###
 
-Sorry, Delphi does not run natively on any other operating systems.  However it should be possible to install Delphi into a virtual machine which is running Windows.
+Delphi does not run natively in non-Windows based operating systems.  However, it is possible to install Delphi into a virtual machine which is running Windows.  If utilizing a Virtual Machine it is possible for Delphi to function.
```

Comment:
I'll work on this, it is a clumsy statement at best and I see another minor issue I want to address as well.